lib/FusionInventory/Agent/Task/Inventory/Linux/Drives.pm

```diff
@@ -36,7 +36,7 @@ sub _getFilesystems {
     # get filesystems list
     my @filesystems =
         # exclude virtual file systems
-        grep { $_->{FILESYSTEM} !~ /^(tmpfs|devtmpfs|usbfs|proc|devpts|devshm|udev)$/ }
+        grep { $_->{FILESYSTEM} !~ /^(tmpfs|devtmpfs|usbfs|proc|devpts|devshm|udev|overlay)$/ }
```

Comment:
Can you confirm the docker volumn is always "overlay" ? If yes, can you test the following:
```suggestion
        grep { $_->{FILESYSTEM} !~ /^(tmpfs|devtmpfs|usbfs|proc|devpts|devshm|udev)$/ && $_->{VOLUMN} !~ /^overlay$/ }
```